projects/hslayers-sensors/src/components/sensors/sensors-unit-list-item.component.ts

```diff
@@ -57,16 +58,29 @@ export class HsSensorsUnitListItemComponent {
    * displaying charts or reopen already existing one.
    */
   sensorToggleSelected(sensor): void {
-    this.hsSensorsUnitDialogService.get(this.app).unit = this.unit;
     sensor.checked = !sensor.checked;
+    const unitDialogServiceRef = this.hsSensorsUnitDialogService.get(this.app);
+    if (unitDialogServiceRef.comparisonAllowed) {
+      //If the opened sensor belongs to unit thats not included add it
```

Comment:
```suggestion
      //If the opened sensor belongs to unit that's not included add it
```
missing end of the sentence